Library/Tests/ViewModels/RewardPledgeViewModelTests.swift

```diff
@@ -40,6 +40,7 @@ internal final class RewardPledgeViewModelTests: TestCase {
   fileprivate let differentPaymentMethodButtonHidden = TestObserver<Bool, NoError>()
   fileprivate let dismissViewController = TestObserver<(), NoError>()
   fileprivate let estimatedDeliveryDateLabelText = TestObserver<String, NoError>()
+  private let estimatedFulfillmentStackViewHidden = TestObserver<Bool, NoError>()
```

Comment:
i have been switching to `private` because we talked about correcting these in the future. I see what you mean about aesthetics though. I'm not quite sure if changing one means I should change them all... :/